Description: Fix the aquire => acquire typo everywhere it exists
Author: Didier 'OdyX' Raboud <odyx@debian.org>
Origin: vendor
Bug: https://github.com/daniel-dressler/ippusbxd/pull/7
Last-Update: 2015-02-11
--- a/src/ippusbxd.c
+++ b/src/ippusbxd.c
@@ -61,13 +61,13 @@
 				ERR_AND_EXIT("Got null packet from tcp");
 			}
 			if (usb == NULL) {
-				usb = usb_conn_aquire(arg->usb_sock, 1);
+				usb = usb_conn_acquire(arg->usb_sock, 1);
 				if (usb == NULL) {
-					ERR("Failed to aquire usb interface");
+					ERR("Failed to acquire usb interface");
 					packet_free(pkt);
 					goto cleanup_subconn;
 				}
-				NOTE("Interface #%d: aquired usb conn",
+				NOTE("Interface #%d: acquired usb conn",
 						usb->interface_index);
 			}
 
--- a/src/usb.c
+++ b/src/usb.c
@@ -195,7 +195,7 @@
 	                                      (uint8_t)selected_config,
 	                                      &config);
 	if (status != 0 || config == NULL) {
-		ERR("Failed to aquire config descriptor");
+		ERR("Failed to acquire config descriptor");
 		goto error;
 	}
 
@@ -452,7 +452,7 @@
 	return staled;
 }
 
-struct usb_conn_t *usb_conn_aquire(struct usb_sock_t *usb,
+struct usb_conn_t *usb_conn_acquire(struct usb_sock_t *usb,
                                    int is_high_priority)
 {
 	int used_high_priority = 0;
--- a/src/usb.h
+++ b/src/usb.h
@@ -63,7 +63,7 @@
 int usb_can_callback(struct usb_sock_t *);
 void usb_register_callback(struct usb_sock_t *);
 
-struct usb_conn_t *usb_conn_aquire(struct usb_sock_t *, int);
+struct usb_conn_t *usb_conn_acquire(struct usb_sock_t *, int);
 void usb_conn_release(struct usb_conn_t *);
 
 void usb_conn_packet_send(struct usb_conn_t *, struct http_packet_t *);
